### PR TITLE
Move client.Call to proto.Call.

### PR DESCRIPTION
**Pull request by petermattis**
This reduces the surface area of the "client" package and also makes
logical sense to place the Call struct in the same package as Request
and Response.

### CODE REVIEW

File: proto/call.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
+// Copyright 2014 The Cockroach Authors.
```
Comment (petermattis) [inline proto/call.go]: This file was moved from `client/call.go` to `proto/call.go` with some edits to adjust for the package movement and adding the `Call` suffix to the methods. Github is showing it as brand new code, but it isn't.

Comment (tamird) [inline proto/call.go]: can you do the move in one commit and the changes in another? that'll make review easier.

Comment (petermattis) [inline proto/call.go]: Sure. Hold on.

Comment (petermattis) [inline proto/call.go]: PTAL

**Comment by tamird** [comment]
LGTM. Really feeling the need to split `proto` up...

**Comment by tschottdorf** [comment]
LGTM.

It makes sense to have each commit pass tests/compile individually (otherwise you break `git bisect`), so it makes sense to squash this right before you merge.

**Comment by tschottdorf** [comment]
How hard is it to split the protos up, btw? Would be a dream.

**Comment by petermattis** [comment]
Ok, going to squash these commits again before merging.

**Comment by tamird** [comment]
@tschottdorf currently blocked on #1354 which is blocked on https://github.com/gogo/protobuf/pull/67. cc @awalterschulze

**Pull request merged**